Mac-App/Frontend/Frontend/Views/Common/ActivityIndicator.swift

```diff
@@ -3,7 +3,7 @@ import SwiftUI
 struct ActivityIndicator: NSViewRepresentable {
     typealias NSViewType = NSProgressIndicator
 
-    @Binding var isAnimating: Bool
+    @State var isAnimating: Bool
     let style: NSProgressIndicator.Style
```

Comment:
Is State better in a sense that UI is auto rendered whenever it changes and you don't need to do explicit binding?

Reply:
This is an outdated diff. Originally I thought `@State` was a better fit for this because flip between true/false was supposed to be done internally, but then after reading this nice guide - https://jaredsinclair.com/2020/05/07/swiftui-cheat-sheet.html 

I learned that `@Biding` was the correct choice as it uses a @State-wrapped property of an ancestor view.